cob_obstacle_distance/launch/example_arm_right.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+    <node name="cob_obstacle_distance" pkg="cob_obstacle_distance" type="cob_obstacle_distance" output="screen">
+        <rosparam param="joint_names">[arm_right_1_joint, arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint, arm_right_6_joint, arm_right_7_joint]</rosparam>
+        <rosparam param="robo_namespace">arm_right</rosparam>
+        <rosparam param="chain_base_link">arm_right_base_link</rosparam>
+        <rosparam param="chain_tip_link">arm_right_7_link</rosparam>
```

Comment:
Instead of ```robo_namespae``` as a parameter, you can launch add a ```ns="arm_right"``` to the ```<node>``` tag and then use ```nh_.getNamespace()``` for retrieving "arm_right"....

All the other parameters are already available on the ParameterServer:
 - ```/arm_right/joint_names```

As for ```chain_base_link``` and ``chain_tip_link``` they can be found here:
 - ```/arm_right/twist_controller/chain_base_link``` and ```/arm_right/frame_tracker/chain_base_link```
 - ```/arm_right/twist_controller/chain_tip_link``` and ```/arm_right/frame_tracker/chain_tip_link```
In order to not define it a third time, we could think about moving them "one level up" and use them from 
 - ```/arm_right/chain_base_link``` and ```/arm_right/chain_base_link```
in all three cases...
 - ```/arm_right/twist_controller/